extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,251 @@
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name."
+        },
+        "xdm:displaycampaignId": {
+          "title": "Display Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the display advertising platform."
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the search advertising platform."
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform."
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform."
+        },
+        "xdm:displayadId": {
+          "title": "Display Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the display advertising platform."
+        },
+        "xdm:adId": {
+          "title": "Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the search advertising platform."
+        },
+        "xdm:term": {
+          "title": "Search Term",
+          "type": "string",
+          "description": "Search Term that contributed to the conversion."
+        },
+        "xdm:termId": {
+          "title": "Search Term Identifier",
+          "type": "string",
+          "description": "Search Term ID on the search advertising platform."
+        },
+        "xdm:matchType": {
+          "title": "Term Match Type",
+          "type": "string",
+          "description": "Match Type associated to the Search Term being targeted for the Ad.",
+          "enum": [
+            "Broad",
+            "Phrase",
+            "Exact",
+            "Advanced",
+            "Standard",
+            "Website",
+            "Product Target",
+            "Product Group",
+            "unknown"
+          ],
+          "meta:enum": {
+            "Broad": "Broad",
+            "Phrase": "Phrase",
+            "Exact": "Exact",
+            "Advanced": "Advanced",
+            "Standard": "Standard",
+            "Website": "Website",
+            "Product Target": "Product Target",
+            "Product Group": "Product Group",
+            "null": "Unknown"
+          }
+        },
+        "xdm:domainPlacement": {
+          "title": "Domain Placement",
+          "type": "string",
+          "description": "Placement Domain"
+        },
+        "xdm:referrerUrl": {
+          "title": "Referrer URL",
+          "type": "string",
+          "description": "The referrer is the webpage that sends visitors to your site using a link."
+        },
+        "xdm:searchTerm": {
```

Comment:
What is the different between searchTerm and term? Is this user entered search term where as the term is the what network used to match to an ad?